global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "3.2.0"
   },
   "sdk": {
-    "version": "7.0.101",
+    "version": "8.0.101",
```

Comment:
Please revert.  Until the Azure SDK team completes testing and rolls the SDK version forward, you'll need to use the approved toolset.

Reply:
Reverted the changes in global.json